sunkit_spex/models/photon/thermal/tests/test_thermal.py

```diff
@@ -0,0 +1,406 @@
+import warnings
+
+import numpy as np
+import pytest
+
+import astropy.units as u
+
+from sunkit_spex import thermal
```

Comment:
Does this need to be `from sunkit_spex.legacy import thermal`?

Reply:
I'm happy to approve otherwise!

Reply:
No.  It uses the imports from the top-level `__init__.py` file.  However, I realised that how that was done there was out of date and fixed it.  So I'll now mark this comment as resolved.